media-sound/qmmp/qmmp-2.2.3.ebuild

```diff
@@ -49,6 +49,7 @@ RDEPEND="
 	)
 	cddb? ( media-libs/libcddb )
 	curl? ( net-misc/curl )
+	doc? ( app-text/doxygen )
```

Comment:
move to BDEPEND (executed during the build)

Reply:
You will be less bothered with the alphabetical order in **B**DEPEND, as it's not libraries/headers ;-)

Reply:
should be **B**DEPEND as it's a tool launched for building.